ERCS/erc_draft_metadata_management_for_nfts.md

```diff
@@ -0,0 +1,140 @@
+---
```

Comment:
```suggestion
---
eip: 7646
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.